api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -274,6 +274,32 @@ public OrderGroup getOrderGroupByUuid(String uuid) throws DAOException {
 	public OrderGroup getOrderGroupById(Integer orderGroupId) throws DAOException {
 		return (OrderGroup) sessionFactory.getCurrentSession().get(OrderGroup.class, orderGroupId);
 	}
+
+	/**
+	 * @see OrderDAO#getOrderGroupsByPatient(Patient)
+	 */
+	@Override
+	public List<OrderGroup> getOrderGroupsByPatient(Patient patient) throws APIException {
+		if (patient == null) {
+			throw new APIException("Patient Cannot Be Null");
```

Comment:
Is there any reason why you are capitalising the first letter of each word in the exception message?